src/components/ErrorPage/ErrorPage.js

```diff
@@ -16,25 +16,30 @@ import {
 const errorTypes = {
   offerNotExist: {
     icon: Close,
-    description: 'Offer does not exist or is not provided.'
+    description: 'Offer does not exist or is not provided.',
+    translationKey: 'offer-error.not-exist'
   },
   generalError: {
     icon: Warning,
-    description: 'Whoops'
+    description: 'Whoops',
+    translationKey: 'whoops'
   },
   alreadyHaveAccess: {
     icon: Lock,
     description:
-      'Good news! Your account already gives you access to the content that comes with this plan.'
+      'Good news! Your account already gives you access to the content that comes with this plan.',
+    translationKey: 'offer-error.access-granted2'
```

Comment:
I dont see any usage of `offer-error.access-granted`. Do we really need this `2` here?

Reply:
![image](https://github.com/Cleeng/mediastore-sdk/assets/68298935/e41cd4cf-614c-4beb-99e9-c35352f87bc3)

it's in offer container